helm/chaos-mesh/templates/controller-manager-deployment.yaml

```diff
@@ -54,6 +54,14 @@ spec:
             value: !!str {{ .Values.chaosDaemon.grpcPort }}
           - name: BPFKI_PORT
             value: !!str {{ .Values.bpfki.grpcPort }}
+          {{- if .Values.controllerManager.allowedNamespaces }}
+          - name: ALLOWED_NAMESPACES
+            value: {{ .Values.controllerManager.allowedNamespaces }}
+          {{- end }}
+          {{- if .Values.controllerManager.ignoredNamespaces }}
+          - name: IGNORED_NAMESPACES
+            value: {{ .Values.controllerManager.ignoredNamespaces }}
+          {{- end }}
```

Comment:
I don' t like using env vars to pass config and I prefer using two flags.
Using env var lacks sth because you cannot find this config when you use `./controller-manager -h`.

Reply:
> I don' t like using env vars to pass config and I prefer using two flags.
> Using env var lacks sth because you cannot find this config when you use `./controller-manager -h`.

I think this is just a configuration style difference. If we need a consistent configuration style, we can reduce the use of environment variables as configurations